src/index.html

```diff
@@ -8,6 +8,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="wrapper">
+      <header class="header"></header>
+      <main class="main">
+        <div class="main__block block">
+          <div class="block__stars stars stars--0">
```

Comment:
Remove unused classes and divs
```suggestion
          <div class="stars stars--0">
```